src/rnp/rnpcfg.h

```diff
@@ -32,10 +32,13 @@
 /* cfg variables known by rnp */
 #define CFG_OVERWRITE "overwrite" /* overwrite output file if it is already exist or fail */
 #define CFG_ARMOUR "armour"       /* armour output data or not */
-#define CFG_DETACHED "detached"   /* produce the detached signature */
-#define CFG_OUTFILE "outfile"     /* name/path of the output file */
-#define CFG_RESULTS "results"     /* name/path for results, not used right now */
-#define CFG_MAXALLOC "maxalloc"   /* maximum memory allocation during the reading from stdin */
+#define CFG_ARMOUR_DATA_TYPE                                                      \
+    "armour_type"               /* armour data type, used with ``enrmour`` option \
```

Comment:
enarmor, not enrmour?

Reply:
same thing apparently

Reply:
I meant missed letter 'a' in the word :)

Reply:
argh... I see :)